test/scim_parser_test.exs

```diff
@@ -0,0 +1,62 @@
+defmodule ScimParserTest do
+  use ExUnit.Case
+
+  @filter_rules [
+    # filter rules (from spec)
+    ~s|userName Eq "john"|,
```

Comment:
removed `(meta.resourceType eq User) or (meta.resourceType eq Group)` as it's actually invalid, the matching value needs to be valid json string, i.e. quoted